source/includes/_chat_summary.md

```diff
@@ -0,0 +1,174 @@
+## Chat Summary
+
+### Resource Overview
+
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/chat_summary/[gmaid]?[query_params]|
+
+### API Name: Chat Summary
+
+### Summary
+
+This API can be used by customers who have our chat widget on their site. The API has metrics on the chat along with traffic information (Paid Campaign/Organic).
+
+### Usage
+Use GET to retrieve information from the Chat Summary API. Data can be returned for a GMAID by a specific date range determined by start date and end date.
+
+Subtotals will be returned for each Master Campaign ID with chats in that date range.
+
+Subtotals can be further aggregated given an interval, and will be totaled for `day`, `calendar_week`, or `calendar_month`. The first day of the interval will be used to denote the interval totals.
+
+Use the master_campaign_id parameter to limit the results to a single Master Campaign ID.
+
+### Parameters
+
+When using the GET method, the results can be filtered using these parameters:
+
+| Param | Required | Function |
+|---|---|---|
+|`start_date`|Yes|Restricts the results to those occurring on or after this date.|
+|`end_date`|Yes|Restricts the results to those occurring on or before this date.|
+|`gmaid`|Yes|Restrict results to the specified Global Master Advertiser ID.|
+|`global_master_campaign_id[]`|No|Restrict results to the specified master campaign id.|
+|`interval_size`|No|Return subtotals for the given intervals within the start-end time provided. Accepts `calendar_month`, `calendar_week`, or `day`.|
+
+### Response Data Details
+
+| Field Name | Datatype | Description |
+|---|---|---|
+|`api_name`|String|Name of the API.|
+|`api_run_date`|String|Date report was run.|
+|`start_date`|String|Start date of report.|
+|`end_date`|String|End date of report.|
+|`time_zone`|String|Time Zone.|
+|`interval_size`|String|Interval Size ('none', 'day', 'calendar_week' or 'calendar_month').|
+|`currency`|String|Current of report.|
```

Comment:
Current should be "currency type of monetary values returned in report"
Which begs the question, are there any monetary values returned in the report?  Cost per X or budget?

Reply:
This has been updated to "Currency Type of Monetary Values Returned in Report (Example: USD, AUD, CAD)", however there are not monetary values returned in the report.